CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+* Make sure the [Process Community Group](https://www.w3.org/community/w3process/) adds the issue to its agenda. This will allow you to explain why you think this is an improvement to the operation of the W3C (it's probably not helpful to provide a Pull Request until you have consensus on the approach).
+
+The best way to ensure your issue gets added to the [Process Community Group](https://www.w3.org/community/w3process/) agenda is to add yourself to the group, please visit [the group homepage](https://www.w3.org/community/w3process/) and select the button "Join or Leave this Group".
+
+Another way to ensure the [Process Community Group](https://www.w3.org/community/w3process/) addressed your issue is to check back on the [issues page](/w3c/w3process/issues) page and ask whether it has been addressed. It is more effective to join the group, but you have this option if you need it. 
+
+The [Process Community Group](https://www.w3.org/community/w3process/) will discuss your issue, and decide whether the change can be made or whether the issue needs to be brought to the [Advisory Board](https://www.w3.org/2002/ab/) or [Advisory Committee](https://www.w3.org/participate/meetings) for discussion; see the next section for more information on this process.
+
+If the [Process Community Group](https://www.w3.org/community/w3process/) agrees the issue does not change the policy and agrees to make the changes detailed in your issue, you may either raise a `pull request` or wait for the Editor to make the required changes. Note: raising a `pull request` **will** be faster. See more details about `pull requests` below.
+
+## Changes that Affect the Way the W3C Operates and Change the Policy
+For changes that represent a change of policy in the W3C, the idea will need to be brought to the [Advisory Board](https://www.w3.org/2002/ab/) or [Advisory Committee](https://www.w3.org/participate/meetings). Again, a Pull Request can wait until you have gained consensus with these two committees.
+
+Getting something in front of both the [Advisory Board](https://www.w3.org/2002/ab/) or [Advisory Committee](https://www.w3.org/participate/meetings) can be tricky. If your organisation is a W3C member please let your Advisory Committee member know; they will then be able to email the Advisory Committee or bring the issue up at an Advisory Committee meeting. To reach the [Advisory Board](https://www.w3.org/2002/ab/) you can [raise an issue](/w3c/w3process/issues) on this repo (at least two AB members monitor this repo) or send an email to <#>.
+
+Gaining consensus from the [Advisory Board](https://www.w3.org/2002/ab/) or [Advisory Committee](https://www.w3.org/participate/meetings) will take some time. After consensus has been reached you may either raise a `pull request` which include the changes agreed by these committees or wait for the Editor to make the required changes. See more details about `pull requests` below.
```

Comment:
https://w3c.github.io/w3process/#AC might be a better link to the Advisory Committee?